fuel-tests/tests/coin.rs

```diff
@@ -219,15 +225,16 @@ async fn only_unspent_coins() {
     assert!(coins
         .results
         .into_iter()
-        .all(|c| c.status == SchemeCoinStatus::Unspent));
+        .all(|c| asset_id == c.asset_id.into()));
```

Comment:
The test is called `only_unspent_coins`, and this was the only assertion where the spent status is checked. Why is this check being removed?

Reply:
That shouldn't have been removed, I've since added it back in. Thank you for catching it